From 1217e739937c90728d57882d9bbe181ddd1cc05a Mon Sep 17 00:00:00 2001
Message-Id: <1217e739937c90728d57882d9bbe181ddd1cc05a.1424394676.git.feitong.yi@intel.com>
In-Reply-To: <00f4430a265007e8e2c016cc7926bf530835ea30.1424394676.git.feitong.yi@intel.com>
References: <00f4430a265007e8e2c016cc7926bf530835ea30.1424394676.git.feitong.yi@intel.com>
From: John Harrison <John.C.Harrison@Intel.com>
Date: Thu, 29 Jan 2015 15:36:38 +0000
Subject: [PATCH 54/61] FOR_UPSTREAM [VPG]: drm/i915: Enable GPU scheduler by
 default

Now that all the scheduler patches have been applied, it is safe to enable.

Change-Id: I128042e85a30fca765ce1eb46c837c62dee66089
For: VIZ-1587
For: VIZ-4741
For: GMIN-3638
Signed-off-by: John Harrison <John.C.Harrison@Intel.com>
---
 drivers/gpu/drm/i915/i915_params.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index 86c33dd..ae8f56b 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -58,7 +58,7 @@ struct i915_params i915 __read_mostly = {
 	.gpu_reset_min_alive_period = 0,
 	.memtrack_debug = 1,
 	.enable_intel_adf = 0,
-	.scheduler_override = 1,
+	.scheduler_override = 0,
 };
 module_param_named(limitbw, i915.limitbw, int, 0400);
 MODULE_PARM_DESC(limitbw,
@@ -249,4 +249,4 @@ MODULE_PARM_DESC(memtrack_debug,
 		 "use Memtrack debug capability (0=never, 1=always)");
 
 module_param_named(scheduler_override, i915.scheduler_override, int, 0600);
-MODULE_PARM_DESC(scheduler_override, "Scheduler override mask (0 = none, 1 = direct submission [default])");
+MODULE_PARM_DESC(scheduler_override, "Scheduler override mask (default: 0)");
-- 
1.7.9.5

